src/pages/Clusters.js

```diff
@@ -51,88 +120,171 @@ const Clusters = () => {
+      disks && h(SimpleFlexTable, {
+        rowCount: filteredDisks.length,
+        columns: [
+          {
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
+            cellRenderer: ({ rowIndex }) => {
+              const disk = filteredDisks[rowIndex]
+              const multiple = _.remove({ status: 'Deleting' }, disksByProject[disk.googleProject]).length > 1
+              return h(Fragment, [
+                disk.googleProject,
+                disk.status !== 'Deleting' && multiple && h(TooltipTrigger, {
+                  content: 'This billing project has multiple active persistent disks. Only one will be used.'
+                }, [icon('warning-standard', { style: { marginLeft: '0.25rem', color: colors.warning() } })])
+              ])
+            }
+          },
+          {
+            size: { basis: 120, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'size', onSort: setDiskSort }, ['Size (GB)']),
+            cellRenderer: ({ rowIndex }) => {
+              const disk = filteredDisks[rowIndex]
+              return disk.size
+            }
+          },
+          {
+            size: { basis: 150, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'status', onSort: setDiskSort }, ['Status']),
+            cellRenderer: ({ rowIndex }) => {
+              const disk = filteredDisks[rowIndex]
+              return disk.status
+            }
+          },
+          {
+            size: { basis: 250, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'created', onSort: setDiskSort }, ['Created']),
+            cellRenderer: ({ rowIndex }) => {
+              return makeCompleteDate(filteredDisks[rowIndex].auditInfo.createdDate)
+            }
+          },
+          {
+            size: { basis: 250, grow: 0 },
+            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'accessed', onSort: setDiskSort }, ['Last accessed']),
+            cellRenderer: ({ rowIndex }) => {
+              return makeCompleteDate(filteredDisks[rowIndex].auditInfo.dateAccessed)
+            }
+          },
+          {
+            size: { basis: 250, grow: 0 },
+            headerRenderer: () => {
+              return h(Sortable, { sort: diskSort, field: 'cost', onSort: setDiskSort }, [`Cost / month (${formatUSD(totalDiskCost)} total)`])
+            },
+            cellRenderer: ({ rowIndex }) => {
+              return formatUSD(persistentDiskCostMonthly(filteredDisks[rowIndex]))
+            }
+          },
+          {
+            size: { basis: 50, grow: 0 },
+            headerRenderer: () => null,
+            cellRenderer: ({ rowIndex }) => {
+              const { id, status } = filteredDisks[rowIndex]
+              // TODO PD: there should be some way of identifying which disk is connected to which runtime
```

Comment:
TOL: if we wanted to identify which runtime a disk is attached to that's a little tricky since the runtimes don't have names or IDs visible in this table, but if we just wanted to show which disks were attached vs floating we could do that pretty easily, something like this: 

`          {
            size: { basis: 150, grow: 0 },
            headerRenderer: () => h(Sortable, { sort: diskSort, field: 'attached', onSort: setDiskSort }, ['Attached to Runtime']),
            cellRenderer: ({ rowIndex }) => {
              const { id } = filteredDisks[rowIndex]
              return _.some({ runtimeConfig: { persistentDiskId: id } }, clusters) ? 'Attached' : 'Unattached'
            }
          },`

which might be nice for users to be able to see whether they have floating disks or not. Conversely we could add a 'PD is attached' (or something) column under runtimes to communicate the same thing.